[BE] Move flatbuffer related python C bindings to script_init

qihqi opened the pull request:
Summary:
Extra C binding module for flatbuffer was introduced because
not all dependencies of Pytorch want (or can) bundle in flatbuffer.

However, flatbuffer is in by default now so this separate binding is not longer needed.

Test Plan: existing unit tests

Differential Revision: D44352583

dbort reviewed (approved):
Nice, looks like a good cleanup.

qihqi committed:
[BE] Move flatbuffer related python C bindings to script_init (#97476)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/97476

Extra C binding module for flatbuffer was introduced because
not all dependencies of Pytorch want (or can) bundle in flatbuffer.

However, flatbuffer is in by default now so this separate binding is not longer needed.

Test Plan: existing unit tests

Reviewed By: tugsbayasgalan

Differential Revision: D44352583

fbshipit-source-id: bfcf2f1693a68d7b177edd2be219ae5095b8fca9
